fw/http.c

```diff
@@ -3029,14 +3029,49 @@ tfw_h1_rewrite_purge_to_get(struct sk_buff *head)
 	return -ENOMEM;
 }
 
+/**
+ * Removes n bytes from the beginning of the message, iterating over skb chain if needed.
+ * @return zero on success and error code on error.
+ *
+ * TODO: to be replaced with future version of ss_skb_chop_head_tail() over all skbs of the message
+ */
+static int
+tfw_h1_chop_leading_crlf(struct sk_buff **head, unsigned n)
```

Comment:
Usually we use `unsigned int`. This is the same, but still would be good to keep the code consistent